docs/configuration.md

```diff
@@ -98,6 +98,7 @@ Mandatory parameters are marked as **Required**, which means that they are requi
 | `strategy` | DefaultStrategy | Defines Strategy class to use.
 | `strategy_path` | null | Adds an additional strategy lookup path (must be a directory).
 | `internals.process_throttle_secs` | 5 | **Required.** Set the process throttle. Value in second.
+| `internals.keep_alive_interval` | 60 | Print keepalive message every X seconds. Set to 0 to disable keepalive messages.
```

Comment:
* Let's set it to 0 by default:
```
`internals.keep_alive_interval` | 0 | Print keepalive message every N seconds. Set to 0 to disable keepalive messages. (Disabled by default.)
```
(the value in `config_full.json` can be kept 60, non-default)

* Actially, it's not a keepalive message, the keepalive is used to keep the connection. This has mostly another purpose -- to verify that the bot is still responsive. It's a "heartbeat" message.  So let's name it 'internals.heartbeat_interval'.
* The message should be more neutral, I propose: `Freqtrade heartbeat message.`, or, with PID (not always PIDs are logged, this will help distinguish bot instances): "Freqtrade heartbeat message. PID=NNN"

Reply:
assuming a "regular" log output, we would have freqtrade repeated many times:

```
2019-10-25 19:53:22,105 - freqtrade.freqtradebot - INFO - I am alive.
2019-10-25 19:53:22,105 - freqtrade.freqtradebot - INFO - Freqtrade hearbeat message PID=xxx.
```

that's 3 times freqtrade in just one line ... which is... odd i think.
Apart from that - no issues in showing the PID.

Reply:
Okay, what about
```
2019-10-25 19:53:22,105 - freqtrade.freqtradebot - INFO - Hearbeat message, PID=xxx.
2019-10-25 19:53:26,105 - freqtrade.freqtradebot - INFO - Hearbeat message, PID=xxx.
...
```
?

Reply:
i've taken "Bot heartbeat" now ... message seems ... redundant, it's a message since i can read it anyway ...